packages/nginx/changelog.yml

```
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.3.3"
+  changes:
+    - description: Tidy up of Markdown in Readme.md, including adding link to developer's documentation, if missing.
```

Can we be more specific with the changelog entries? If a link to dev docs wasn't added, we shouldn't mention it.

```suggestion
    - description: Tidy up of Markdown in Readme.md.
```